server/util/claims/claims.go

```diff
@@ -109,6 +109,22 @@ func (c *Claims) IsSAML() bool {
 	return c.SAML
 }
 
+func (c *Claims) AssembleJWT() (string, error) {
```

Comment:
nit: I think this could be a top-level function in the `claims` package - adding it to the struct's public interface limits our ability to restrict the visibility of this function later on

Reply:
That makes this PR just a couple of lines, so I'll just fold it into the next one